tests/_tests_cv_classification_transforms/experiment.py

```diff
@@ -21,10 +21,11 @@ def __getitem__(self, index: int) -> Tuple:
         """
         image, target = self.data[index], self.targets[index]
 
+        dict_ = {"image": image, "targets": target}
         if self.transform is not None:
-            image = self.transform({"image": image})["image"]
+            dict_ = self.transform(dict_)
 
-        return image, target
+        return dict_
```

Comment:
you mean, we could not use `KorniaCallback` without kv data-batches?

Reply:
fixed